server/utils/utils.ts

```diff
@@ -33,6 +33,10 @@ const parseIsoDate = (date: string) => {
   return date ? parse(date, 'yyyy-MM-dd', new Date()) : null
 }
 
+const parseIsoDateTime = (date: string) => {
+  return date ? parse(date, 'yyyy-MM-dd HH:mm:ss', new Date()) : null
```

Comment:
Should we be using the `yyyy-MM-dd'T'HH:mm:ss...` format?

Reply:
have no idea!? will ask devs

Reply:
actually makes sense, will change

Reply:
I think if I reading this correct https://www.npmjs.com/package/date-fns-tz it's supported by default..